### ISSUE TITLE
Model interpretability missing from new Pytorch implemenation

### ISSUE THREAD

**Issue by nick-j-braun** [sindhura97/STraTS#5]
Contribution scores seem to be missing from the latest version of the repo. I-STraTS is built but there is no reference to how to generate the scores shown in the paper. How would one go about obtaining these scores?

**Comment by ealonso-vicomtech** [comment]
Hi, any news on this issue? I'm having the same problem, I don't know how to generate the scores in the new torch implementation.

**Comment by nickjbraun** [comment]
> Hi, any news on this issue? I'm having the same problem, I don't know how to generate the scores in the new torch implementation.

Nope, I successfully trained an istrats model but haven't been able to figure out how to actually get contribution scores
